CHANGELOG.md

```diff
@@ -17,6 +17,10 @@
 
 ## [0.11.1] - 2020-10-23
 
+
+### Fixed
+* Add pass-in pauseState parameter for OnAddTrack() so track can be added for paused streams
```

Comment:
This is sort of confusing and deep in implementation.  Can probably just say something like:
```
Fixed `videoTileDidAdd` not being called for paused tiles.
```

Reply:
New changelog should go to the `## Unreleased` section on the top.